scripts/validate-models-s3.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+strategies=${STRATEGIES:=fast,hi_res,ocr_only}
+input_dir=s3://utic-dev-tech-fixtures/partition-strategy-evaluation/pdf/lang/
+
+function list_s3_dirs() {
+  # s3 doesn't have a notion of directories, need to do some parsing to get them from a particular s3 path
+  path=$1
+  aws s3 ls "$path" | grep -E '/' | awk '{ print substr( $2, 1, length($2)-1 ) }'
+}
+
+function run_inference() {
+  strategy=$1
+  lang=$2
+  input_path="$input_dir$lang"
+  echo "Running inference with strategy $strategy and ocr language $lang on input dir $input_path"
+  output_dir=partition-strategy-evaluation/output/json/$strategy
+  echo "writing output content to local dir $output_dir"
+  PYTHONPATH=. ./unstructured/ingest/main.py \
+    --metadata-exclude filename,file_directory,metadata.data_source.date_processed \
+    --remote-url "$input_path" \
+    --structured-output-dir "$output_dir" \
+    --preserve-downloads \
+    --partition-strategy "$strategy" \
+    --partition-ocr-languages "$lang" \
+    --download-dir partition-strategy-evaluation/input/
```

Comment:
I don’t think we need to keep the downloaded PDFs locally so I don’t think `--preserve-downloads` and `--download-dir` are necessary.

Reply:
Since the script can take a while, this helps in case something breaks along the way.